build/packaging/transforms/Alloy.Sample.BlockEnhancements.web.config.transform

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<modifications>
+    <add path="configuration/episerver.framework/virtualRoles/providers">
+        <add __keyAttributes="name" name="EPiBetaUsers" type="EPiServer.Security.AuthenticatedRole, EPiServer.Framework" />
+    </add>
+    <add path="configuration/episerver.framework">
+        <licensing licenseFilePath="E:\License.config" />
```

Comment:
will this work (absolute path to license)?

Reply:
This is just the "daily site" system we use to host a new site for every git branch for ease of reviewing internally.